src/runtime/pkg/oci/utils.go

```diff
@@ -1000,6 +1003,12 @@ func addRuntimeConfigOverrides(ocispec specs.Spec, sbConfig *vc.SandboxConfig, r
 		return err
 	}
 
+	if err := newAnnotationConfiguration(ocispec, vcAnnotations.ForceGuestPull).setBool(func(forceGuestPull bool) {
```

Comment:
Since you also offer annotations (thanks) you may want to add it to `docs/how-to/how-to-set-sandbox-config-kata.md`

Reply:
done